cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/client/AmazonRdsClient.java

```diff
@@ -47,6 +62,35 @@ public DescribeDBInstancesResult describeDBInstances(DescribeDBInstancesRequest
         return client.describeDBInstances(describeDBInstancesRequest);
     }
 
+    public boolean doesDbParameterGroupExist(String dbParameterGroupName) {
+        DescribeDBParameterGroupsRequest describeDBParameterGroupsRequest = new DescribeDBParameterGroupsRequest()
+                .withDBParameterGroupName(dbParameterGroupName);
+        try {
+            DescribeDBParameterGroupsResult result = client.describeDBParameterGroups(describeDBParameterGroupsRequest);
+            LOGGER.debug("DB parameter group already exists: {}, result: {}", dbParameterGroupName, result);
+            return true;
+        } catch (DBParameterGroupNotFoundException e) {
+            LOGGER.debug("DB parameter group does not exist: {}", dbParameterGroupName);
+            return false;
+        }
+    }
+
+    public DBParameterGroup createParameterGroup(String dbParameterGroupFamily, String dbParameterGroupName) {
+        CreateDBParameterGroupRequest createDBParameterGroupRequest = new CreateDBParameterGroupRequest()
+                .withDBParameterGroupFamily(dbParameterGroupFamily)
+                .withDBParameterGroupName(dbParameterGroupName)
+                .withDescription("Parameter group for enforcing SSL for an RDS");
```

Comment:
this string should come from a parameter instead

Reply:
What do you mean by that, from application.yml? What would be the purpose of that?
Currently, for postgres 10 it is an autogenerated text:
"DB parameter group for dbsvr-98e64356-85ca-4500-a106-03dcaf6547e5"

I am thinking now to put that back actually.

Reply:
no, I meant method parameter, as the method seems to be generic

Reply:
Ok, I moved it to method parameter. The text will be like what is now: 
DB parameter group for dbsvr-c45df68b-0491-4501-863e-104ed35c0fc9